tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+            raise ValueError("model not known: " + model)
+        return set(self.required[model])
+
+    def get_accuracy_target(self, model):
+        if model not in self.accuracy_target:
+            raise ValueError("model not known: " + model)
+        return self.accuracy_target[model]
+
+    def get_performance_sample_count(self, model):
+        if model not in self.performance_sample_count:
+            raise ValueError("model not known: " + model)
+        return self.performance_sample_count[model]
+
+
+def get_args():
+    """Parse commandline."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input", required=True, help="submission directory")
+    parser.add_argument("--version", default="v0.7", choices=list(MODEL_CONFIG.keys()), help="mlperf version")
+    parser.add_argument("--submitter", help="filter to submitter")
+    parser.add_argument("--csv", default="summary.csv", help="csv file with results")
+    args = parser.parse_args()
+    return args
+
+
+def model_map(config, model):
+    """Map models names to the official mlperf name."""
+    if model in config.models:
+        return model
+    if model in MODEL_MAPPING:
+        return  MODEL_MAPPING[model]
+    if model.startswith("mobilenet"):
+        model = "mobilenet"
+    elif model.startswith("rcnn"):
+        model = "ssd-small"
+    elif model.startswith("ssdlite") or model.startswith("ssd-inception") or model.startswith("yolo") or \
+            model.startswith("ssd-mobilenet") or model.startswith("ssd-resnet50"):
+        model = "ssd-small"
+    return model
+
+
+def list_dir(*path):
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isdir(os.path.join(path, f))]
+
+
+def list_files(*path):
+    path = os.path.join(*path)
+    return [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+
+
+def split_path(m):
+    return m.replace("\\", "/").split("/")
+
+
+def ignore_errors(line):
+    if "check for ERROR in detailed" in line:
+        return True
+    if "Loadgen built with uncommitted changes" in line:
+        return True
+    if "Ran out of generated queries to issue before the minimum query count and test duration were reached" in line:
+        return True
+    if "CAS failed":
+        return True
+    return False
+
+
+def check_accuracy_dir(config, model, dir):
+    is_valid = False
+    acc = None
+    model_norm = model_map(config, model)
+    acc_type, acc_target = config.get_accuracy_target(model_norm)
+    if not isinstance(acc_target, list):
+        acc_target = [acc_target]
+    acc_target = list(sorted(acc_target, reverse=True))
+    pattern = ACC_PATTERN[acc_type]
+    with open(os.path.join(dir, "accuracy.txt"), "r") as f:
+        for line in f:
+            m = re.match(pattern, line)
+            if m:
+                is_valid = True
+                acc = m.group(1)
+                break
+
+    if acc:
+        for a in acc_target:
```

Comment:
I don't get this part. How does it enforce 99.9% accuracy target for the logs that correspond to high-accuracy?

Reply:
yeah, I now removed the accuracy list and expect a single accuracy given by a model 'bert-99' or 'bert-99.9'